releases/edge/mac-manifests.json

```diff
@@ -1,6 +1,16 @@
 {
     "app_name": "JackTrip",
     "releases": [
+        {
+            "version": "1.6.2-rc.3",
+            "changelog": "Ability to open app via URL schemes, displaying latency stats, and Virtual Studio device support. Learn more here: https://github.com/jacktrip/jacktrip/releases/tag/v1.6.2-rc1",
```

Comment:
Shouldn't this be "(...)releases/tag/v1.6.2-rc**3**"?

Reply:
Yes, corrected